emulator/src/extensions/touch/ext_touch.c

```diff
@@ -248,25 +249,40 @@ static bool touchInit(emuArgs_t* emuArgs)
     if (emuArgs->emulateTouch)
     {
         requestPane(&touchEmuCallback, PANE_BOTTOM, PANE_MIN_SIZE, PANE_MIN_SIZE);
-        return true;
-    }
-    else
-    {
-        return false;
     }
+
+    return true;
 }
 
 static int32_t touchKey(uint32_t key, bool down)
 {
-    const int32_t phiMap[] = {90, 0, 180, 270};
+    // Map from state to touchpad rotation. Button bits are in URLD order
+    const int32_t phiMap[] = {
```

Comment:
Smart. I did something similar last year with the five touch areas on the strip.

Reply:
That's where I got the idea! 😁